server/test/unit/plugins/test_importer.py

```diff
@@ -1,15 +1,62 @@
 # -*- coding: utf-8 -*-
 
-import unittest
+from unittest import TestCase
 
-import mock
+from mock import patch, Mock
 
 from pulp.plugins.importer import Importer
 
 
-class TestImporter(unittest.TestCase):
+class TestImporter(TestCase):
 
-    @mock.patch('pulp.plugins.importer.sys.exit', autospec=True)
+    @patch('pulp.plugins.importer.LocalFileDownloader')
+    @patch('pulp.plugins.importer.importer_config_to_nectar_config')
+    def test_get_local_downloader(self, to_nectar, local):
+        url = 'file:///martin.com/test'
+        config = Mock()
+
+        # test
+        downloader = Importer.get_downloader(config, url)
+
+        # validation
+        to_nectar.assert_called_once_with(config.flatten.return_value)
+        local.assert_called_once_with(to_nectar.return_value)
+        self.assertEqual(downloader, local.return_value)
+
+    @patch('pulp.plugins.importer.HTTPThreadedDownloader')
+    @patch('pulp.plugins.importer.importer_config_to_nectar_config')
+    def test_get_http_downloader(self, to_nectar, http):
+        url = 'http:///martin.com/test'
```

Comment:
/\/\/\//\/\//

Maybe that's not the easiest way to say http:/// should probably be http://